### ISSUE TITLE
Feature request - Create button for multiple SSH host

### ISSUE THREAD

**Issue by DRINGOT** [zobnin/aio_ssh_button_plugin#1]
Hello,

I was wondering if it would be possible to add a feature that allows us to create buttons for multiple SSH hosts? Currently, we can only execute commands for one host at a time. It would be useful if we could simply click a button to add a different host and assign the commands to the desired host.

Would it be possible to implement this feature in the future? This would be really helpful for those of us who need to connect to multiple SSH hosts regularly.

Thank you very much!

**Comment by zobnin** [comment]
Yes, the plugin needs many improvements, including this one. But I don't have time to do that yet.
